src/reference/asciidoc/webflux.adoc

```diff
@@ -319,3 +319,8 @@ See <<./http.adoc#http-outbound,HTTP Outbound Components>> for more possible con
 
 Since WebFlux components are fully based on the HTTP protocol, there is no difference in the HTTP headers mapping.
 See <<./http.adoc#http-header-mapping,HTTP Header Mappings>> for more possible options and components to use for mapping headers.
+
+[[webflux-request-attribute]]
+=== WebFlux Request Attribute
+
+Starting with Version 6, Webflux outbound request can have request attribute which can be specified in `attribute-variables-expression` attribute. This attribute will take an `expression` that should be evaluated in `Map`.
```

Comment:
"Version" word not capitalized.
Version number is `6.0`.
One sentence per line rule, please: https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line.

I wouldn't imply an XML config. Better to say `WebFluxRequestExecutingMessageHandler` and `setAttributeVariablesExpression()`. 
The XML configuration is really left in a maintenance mode and better to move away from it, more over with modern trends about native images.

Would be great to have some sample configuration and the reasoning behind this feature.

Reply:
Request attribute is standard webclient feature which can be used in multiple ways. Sample configuration (https://github.com/saxenaj/spring-integration/blob/GH-3936/spring-integration-webflux/src/test/java/org/springframework/integration/webflux/config/WebFluxOutboundGatewayParserTests-context.xml#L43) and implementation (https://github.com/saxenaj/spring-integration/blob/GH-3936/spring-integration-webflux/src/test/java/org/springframework/integration/webflux/outbound/WebFluxRequestExecutingMessageHandlerTests.java#L391) are there in test cases.